metadata.json

```diff
@@ -26,7 +26,7 @@
   "requirements": [
     {
       "name": "pe",
-      "version_requirement": ">=3.7.0 <4.0.0"
+      "version_requirement": ">=3.0.0 <=4.3.1"
```

Comment:
For PE, now the PE agent and the POSS agent are the same this is generally not needed. I believe we're thinking about deprecating this field entirely as it can cause some confusion for modules.